content/en/blog/_posts/2019-06-29-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: 'Some APIs Are Being Deprecated: Here's What You Need To Know
+date: 2019-06-29
+---
+
+**Author**: Vallery Lancey (Lyft)
+
+As part of evolving the Kubernetes API, APIs are periodically
+re-organized or upgraded. When this happens, the old API is deprecated, and then
+eventually removed.
+
+There are **4** sets of resources with APIs that are deprecated, will be stop
+being served in the coming months. If you use any of these, this post is for you:
+
+* NetworkPolicy
+* PodSecurityPolicy
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet
+* Ingress
+
+None of these resources are being removed from Kubernetes. NetworkPolicies,
+Deployments, and so on will still remain part of Kubernetes, and are not being
+deprecated in any way. However, to continue using these resources, you must
+use a current version of the Kubernetes API.
+
+# Migration Details
+
+* NetworkPolicy: will no longer be served from **extensions/v1beta1** in **v1.16**.
+  * Migrate to the networking.k8s.io/v1 API, available since v1.8. Existing persisted
+    data can be retrieved/updated via the networking.k8s.io/v1 API.
+* PodSecurityPolicy: will no longer be served from **extensions/v1beta1** in **v1.16**.
+  * Migrate to the policy/v1beta1 API, available since v1.10. Existing persisted
+    data can be retrieved/updated via the policy/v1beta1 API.
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet:  will no longer be served
+from **extensions/v1beta1**, **apps/v1beta1**, or **apps/v1beta2** in **v1.16**.
+  * Migrate to the apps/v1 API, available since v1.9. Existing persisted data
+    can be retrieved/updated via the apps/v1 API.
+* Ingress: will no longer be served from **extensions/v1beta1** in **v1.18**.
+  * Migrate to the networking.k8s.io/v1beta1 API. Existing persisted data can be
+    retrieved/updated via the networking.k8s.io/v1beta1 API.
+
+# What To Do
+
+Kubernetes 1.16 is due to be released in September 2019, so be sure to audit
+your configuration and integrations now!
+
+* Change YAML files to use the newer APIs.
```

Comment:
Because "use" is fairly generic, would it be more specifically accurate to say "call" instead? For example:

> * Update YAML files to call the newer APIs